packages/angular/src/generators/ng-add/files/root/nx.json__tmpl__

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
It's not useless, it's used. There were (are) two places where this file is generated. I didn't refactor that into a single place because this PR was for moving the generator, not for refactoring it. I'll be refactoring this generator next, it needs way more changes than the ones mentioned here.

That said, I prefer having it in a template, that's what the templating is for when it comes to generators and that's what we use for all new files, which is the case here. I don't see why it'd be better to do it in the generator code and why this file would be different than other files that can't be done in the template (we use the template in the workspace generator as well).